README.md

```diff
@@ -139,7 +139,15 @@ Run `python src/py_sonar_scanner`
 
 # Run the tests
 
-Run `poetry install` to install the dependencies
+Run `poetry install` to install the dependencies. By default, the dependencies are installed from a private repository. 
+
+If you wish to install the dependencies from the public PyPI repository, remove the following from `pyproject.toml`:
+```toml
+[[tool.poetry.source]]
+name = "repox"
+url = "https://repox.jfrog.io/artifactory/api/pypi/sonarsource-pypi/simple/"
+priority = "default"
+```
```

Comment:
I think this is quite nice, but I guess the main point here would be to explain how to setup the authentication to our repox for poetry, this is [explained here](https://python-poetry.org/docs/repositories/) and maybe the local setup should be explained by setting:
```
export POETRY_HTTP_BASIC_PYPI_USERNAME=<username>
export POETRY_HTTP_BASIC_PYPI_PASSWORD=<password> 
```
I guess just a quick sentence as well explaining the PASSWORD should be the api token created on your account in repox.